blocks/editable/index.js

```diff
@@ -212,6 +212,42 @@ export default class Editable extends wp.element.Component {
 		if ( keyCode === BACKSPACE ) {
 			this.onSelectionChange();
 		}
+
+		if ( keyCode === ENTER && this.props.inline && this.props.onSplit ) {
+			const endNode = this.editor.selection.getEnd();
+
+			if ( endNode.nodeName !== 'BR' ) {
```

Comment:
Can we add some clarifying inline comments here? Not sure why we care to test whether the final node is a linebreak, or why we're expecting a second line break per the subsequent condition.

__Edit:__ After testing, I understand now that we want two line breaks before split.